### ISSUE TITLE
My first plugin

### ISSUE THREAD

**Issue by Miesjh** [MscrmTools/XrmToolBox#711]
Just created my first XrmToolBox plugin; https://www.nuget.org/packages/XrmToolBox.Dynamics365V9JavascriptValidator/

But I cannot find it in the XrmToolBox Plugins Store.

What could it be? Dependency is XrmToolBoxPackage.

**Comment by MscrmTools** [comment]
Should be XrmToolBox in the nuget package

**Comment by Miesjh** [comment]
When I look at other plugins they all have a dependency with: 
https://www.nuget.org/packages/XrmToolBox/

But I cannot open the page. 404. Also this package is not visible.

**Comment by shytikov** [comment]
That's the trick :) You're referencing none-existing package and that dependency is used as criteria.

**Comment by Miesjh** [comment]
Ok. I changed that. Now waiting for Publishing and indexing. Thanks for the quick reply.

**Comment by Miesjh** [comment]
Package is Published and Indexed, but still not visible.

**Comment by rappen** [comment]
Have you added it here as well? https://www.xrmtoolbox.com/plugins/
Until it is synced there, it should be visible if you switch the Plugins Store to old legacy style.

**Comment by Miesjh** [comment]
I registered. 

Now I'm getting this error:

Va;lueType 'System.DateTime' cannot be null, when opening the store.

**Comment by shytikov** [comment]
Same for me, BTW...

**Comment by MscrmTools** [comment]
I'm on it guys...

**Comment by MscrmTools** [comment]
Fixed

**Comment by MscrmTools** [comment]
@Miesjh your plugin control should handle resizing. You must dock the Log groupbox to fill remaining space

**Comment by shytikov** [comment]
I had issues with sorting in plugin store also (basically order was random for download count columns)

**Comment by MscrmTools** [comment]
@shytikov please open a separate issue for that point, please

**Comment by shytikov** [comment]
@MscrmTools of course! I was trying to come up with steps to reproduce, and how I discovered that error, and that sentence was part of it. And then I discovered that issue was already fixed while I was typing...

I felt so sad so I did stupid work, so decided to post at least one sentence that still was valid :)

**Comment by MscrmTools** [comment]
For information, the problem is about some calculated fields... not calculated yet

**Comment by Miesjh** [comment]
It works! Thanks

**Comment by Miesjh** [comment]
Version 1.0.0.2 is now available. Please approve the plugin.

**Comment by MscrmTools** [comment]
Ok, are you sure you don’t want to provide source code of your plugin in GitHub ?

**Comment by Miesjh** [comment]
That will be the next step.

**Comment by MscrmTools** [comment]
Ok, when it’s done, you can change project url to your GitHub repository
